### ISSUE TITLE
Change speed or viscosity

### ISSUE THREAD

**Issue by yamifr07** [yoavlt/LiquidFloatingActionButton#56]
How can I change the animation speed, without making changes in the lib code?

**Comment by edwardjiang7** [comment]
Yes, look at the code here: `class CircleLiquidBaseView : ActionBarBaseView {`

**Comment by yamifr07** [comment]
Could you help me and put the code? I don't know how to modify it outside the lib

**Comment by edwardjiang7** [comment]
In the LiquidFloatingActionButton.swift, find `class CircleLiquidBaseView : ActionBarBaseView {`. Then change the viscosity from there.

**Comment by ipagong** [comment]
i think no way to change values when use cocoapod...

durations and viscosity values are not open or public properties....
and override ? also impossible...
